plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java

```diff
@@ -53,7 +53,8 @@
 			ResourceKind.BUILD, 
 			ResourceKind.IMAGE_STREAM, 
 			ResourceKind.ROUTE,
-			ResourceKind.TEMPLATE
+			ResourceKind.TEMPLATE,
```

Comment:
We should investigate, but I believe IMAGE_STREAM_TAG is not a watchable resource.  Trying to watch on such a resource should probably throw; we should consider adding an issue to ensure we do that.